build/constants.yaml

```diff
@@ -109,6 +109,10 @@
     final-stretch-dot-net-60-sdk-version: 6.0.301
     final-stretch-dot-net-core-app-60-run-time-version: 6.0.6
     final-stretch-aspnet-core-app-60-run-time-version: 6.0.6
+
+    final-stretch-dot-net-60-sdk-version: 7.0.100-preview.6.22352.1
```

Comment:
can you update these to be `final-stretch-dot-net-70-sdk-version` etc. Having the same name seems to be overwriting the actual .NET 6 constants.

Reply:
Thanks for pointing it out!

Reply:
Makes sense! I'm probably missing something -- I don't remember off the top of my head, but I think for dotnet 7 preview 7, we decided to delete versionsToBuild.txt for stretch after publishing dotnet 7 preview 7 sdks to the dev storage account. I think this is the reason why we are seeing these sdks in the dev storage account.

Reply:
we did that for .NET 3.1 and .NET 6, which is why I released the PR with those final stretch versions. Looks like we did not remove the preview 7 version from [dotnet stretch versionsToBuild.txt](https://github.com/microsoft/Oryx/blob/main/platforms/dotnet/versions/stretch/versionsToBuild.txt#L95), which is why we see it in the dev storage account.